db/migrate/20150320164433_remove_individual_nhsuk_domains_from_whitelist.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveIndividualNhsukDomainsFromWhitelist < ActiveRecord::Migration
+  def up
+    WhitelistedHost.where("hostname LIKE '%.nhs.uk%'").each do |whitelisted_host|
```

Comment:
Strictly we shouldn't have the second `%` here, because we only automatically whitelist domains which end with `.nhs.uk`, rather than ones which contain it.